config.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+    "pluginAlias": "UEBoomSpeaker",
+    "pluginType": "accessory",
+    "schema": {
+        "type": "object",
+        "properties": {
+            "name": {
+                "title": "Name",
+                "type": "string",
+                "required": true,
+                "default": "Bathroom Speaker",
+                "description": "The name that will appear in your homebridge log."
+            },
+            "accessory": {
+                "title": "Accessory",
+                "type": "string",
+                "default": "UEBoomSpeaker",
+                "required": true,
+                "description": "This must be UEBoomSpeaker."
+            },
+            "speaker": {
+                "title": "speaker",
```

Comment:
Also, if you could fix the `title` to `Speaker` it would be great.

Reply:
made the Speaker uppercase.